### PR TITLE
Display package capabilities on UI

### PR DESCRIPTION
**Pull request by jyvenugo**
Parse manifest file to obtain capabilities and display the same on the UI.

### CODE REVIEW

File: preview/Win7Msix/Win7MSIXInstaller.sln [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+EndProject
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "Win7MSIXInstallerLib", "Win7MSIXInstallerLib\Win7MSIXInstallerLib.vcxproj", "{46BE19BD-72E0-4452-9919-33A1F8A5F064}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{B0C6E315-832B-4665-B979-4111A8C2F93B}"
+	ProjectSection(SolutionItems) = preProject
+		.editorconfig = .editorconfig
+	EndProjectSection
+EndProject
+Global
+	GlobalSection(SolutionConfigurationPlatforms) = preSolution
```
Comment (wcheng-msft) [inline preview/Win7Msix/Win7MSIXInstaller.sln]: merge or undo any changes for this file, it got renamed

File: lib/zlib/zconf.h [hunk 2]
```diff
@@ -1,534 +0,0 @@
...
-/*
- * If you *really* need a unique prefix for all types and library functions,
- * compile with -DZ_PREFIX. The "standard" zlib should be compiled without it.
- * Even better than compiling with -DZ_PREFIX would be to use configure to set
- * this permanently in zconf.h using "./configure --zprefix".
- */
-#ifdef Z_PREFIX     /* may be set to #if 1 by ./configure */
-#  define Z_PREFIX_SET
-
-/* all linked symbols and init macros */
```
Comment (wcheng-msft) [inline lib/zlib/zconf.h]: undo change for this file. not sure when it's used but it seems to get deleted every time you build msix.dll

File: preview/Win7Msix/Win7MSIXInstaller.sln (hunk 1 above)
Comment (jyvenugo) [reply preview/Win7Msix/Win7MSIXInstaller.sln]: Created new PR #105 to track changes without the changes in this file.

File: lib/zlib/zconf.h (hunk 2 above)
Comment (jyvenugo) [reply lib/zlib/zconf.h]: Created new PR #105 to track changes without the changes in this file.

**Pull request closed without merging**